.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         type: string
         default: '5.2.2'
     executor:
-      name: 'samvera/ruby'
+      name: 'samvera/ruby_fcrepo_solr_redis_postgres'
```

Comment:
I think it should be safe to just have the ruby executor since you shouldn't need to have postgres et al running for bundle install.

Reply:
Cool. I'll poke at the change.